sconstruct

```diff
@@ -331,6 +331,7 @@ for po in env.Glob(sourceDir.path + "/locale/*/lc_messages/*.po"):
 
```

Comment:
please also add a reference in https://github.com/nvaccess/nvda/blob/495dee36e0c5a417ca369ed4ae17115ba5372eec/source/setup.py#L312